src/main/java/org/opentripplanner/api/model/ApiLeg.java

```diff
@@ -74,7 +72,7 @@
      */
     @XmlAttribute
     @JsonSerialize
-    public String mode = TraverseMode.WALK.toString();
+    public String mode = "WALK";
```

Comment:
We should do ApiMode mapping in one place.

Reply:
This is a default value that are set on other fields in this class too. I have now changed it back to how it was, except using the api specific ApiRequestMode enum instead of TraverseMode.